cli/util/cancel.go

```diff
@@ -52,7 +52,7 @@ func cancelTx(ctx *cli.Context) error {
 
 	mainTx, _ := c.GetRawTransactionVerbose(txHash)
 	if mainTx != nil && !mainTx.Blockhash.Equals(util.Uint256{}) {
-		return cli.NewExitError(fmt.Errorf("transaction %s is already accepted at block %s", txHash, mainTx.Blockhash.StringLE()), 1)
+		return cli.NewExitError("Target transaction accepted", 1)
```

Comment:
Do we need to change the message then?

Reply:
The price is once extra RPC call. Fixed.